Packs/DomainTools_Iris/Integrations/DomainTools_Iris/DomainTools_Iris.yml

```diff
@@ -4,13 +4,10 @@ commonfields:
   version: -1
 configuration:
 - display: API Username
```

Comment:
You should leave the parameters `username` and `apikey`, and change to `required: false` and add `hidden: true`, and add a new parameter as follows:
```
- display: API Username
  name: credentials
  type: 9
  required: false
  displaypassword: API Key
```